packages/@guardian/source-react-components/src/text-area/TextArea.tsx

```diff
@@ -131,6 +131,7 @@ export const TextArea = ({
 				required={!optional}
 				rows={rows}
 				className={getClassName()}
+				defaultValue={value}
```

Comment:
Should the name of the prop passed in also be `defaultValue`? It confused me when I read the story as I expected the names to match up as the behaviour is different

Reply:
I think maybe we could add another prop called `defaultValue`, but if we were to change `value` to `defaultValue` we'd be at cross purposes with the way this component is [generally used](https://github.com/guardian/dotcom-rendering/blob/3ac24426c459fb8456c81b5148da6c883872189e/dotcom-rendering/src/web/components/Callout/TextArea.tsx#L23).

That might make sense though?

Reply:
I've realised the issue was actually just with the way the story was composed - if we have a controlled component we need to pass a change handler in, this was missing in the story.

Now its added in we can use the value field `value={value}` and everything is nice.

Reply:
Sounds good 👍